### PR TITLE
Update project

### PR DESCRIPTION
**Pull request by Omicron7**
Switch to nodejs10 and yarn and fix issues.
Use webpack and babel
Use terraform managed role for deployments

This includes the PR to update memory to 512 for campaign import.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -23,6 +32,7 @@ functions:
   campaign:
     handler: campaign/import.handler
     timeout: 300
+    memorySize: 512
```
Comment (wrandall22) [inline serverless.yml]: Can we do 1024 instead?

File: api/gr.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ const createEntity = (user, callback) => {
     }
   }, (e, r, body) => {
     if (e || r.statusCode >= 400) {
-      callback(false)
+      callback(false) // eslint-disable-line
```
Comment (wrandall22) [inline api/gr.js]: Why did this line need to be disabled for eslint?

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  'comments': false,
+  plugins: ['source-map-support'],
```
Comment (wrandall22) [inline babel.config.js]: Was there a reason to put the other two attributes in quotes, or not putting this one in quotes? Seems like they should be consistent with each other.

File: package.json [hunk 4]
```diff
@@ -20,19 +33,22 @@
...
+		"env": {
+			"node": true,
+			"jest": true
+		}
+	},
 	"jest": {
-		"verbose": true,
-		"testEnvironment": "node",
-		"setupTestFrameworkScriptFile": "./test/test-setup.js",
-		"coveragePathIgnorePatterns": [
```
Comment (wrandall22) [inline package.json]: Are these ignores no longer needed?

File: serverless.yml [hunk 5]
```diff
@@ -2,13 +2,22 @@ service: scale-of-belief-import
...
+  stackTags:
+    Name: ${env:PROJECT_NAME}-${self:custom.environmentMap.${env:ENVIRONMENT}}
+    owner: apps@cru.org
+    application: ${env:PROJECT_NAME}
+    env: ${self:custom.environmentMap.${env:ENVIRONMENT}}
+    managed_by: serverless-framework
+    function: lambda
   memorySize: 256
   environment: ${file(serverless/environment.js)}
-  $<<: ${file(serverless/provider.js)}
```
Comment (wrandall22) [inline serverless.yml]: I'm not understanding how the `provider.js` functionality is migrating over. Guessing it has to do with webpack, but not sure.

File: api/gr.js (hunk 2 above)
Comment (Omicron7) [reply api/gr.js]: standardjs error, When you use the name `callback` or `cb` and don't pass and `Error` object as the first param, it gives an error. `standard/no-callback-literal`

File: babel.config.js (hunk 3 above)
Comment (Omicron7) [reply babel.config.js]: Nope, copy paste from other projects.

File: package.json (hunk 4 above)
Comment (Omicron7) [reply package.json]: All of that's handled by `babel-jest` now as it needs to transform files before jest can run them.

File: serverless.yml (hunk 5 above)
Comment (Omicron7) [reply serverless.yml]: I'm removing the `sls-production.yml` provider files from ecs_config. Only 1 project used it and it made things more complicated.

Comment (Omicron7) [reply serverless.yml]: Everything in the sls file was added directly to the project.
https://github.com/CruGlobal/scale-of-belief-import/pull/20/files/d1528aadf4a577be676a1ad6a5b5daaee2c0f1e2#diff-220045f50daee3e35e21b9d5fe19d67eR8-R18

**Pull request merged**